userspace/falco/falco.cpp

```diff
@@ -158,6 +158,8 @@ static void usage()
 	   "                               This causes every single line emitted by falco to be flushed,\n"
 	   "                               which generates higher CPU usage but is useful when piping those outputs\n"
 	   "                               into another process or into a script.\n"
+	   " -u                            Flip the inspector code to parse from userspace. This can be used in conjunction with\n"
```

Comment:
Not sure about the term "Flip". It sounds too specific to me.

WDYT about the following suggestion?

```suggestion
	   " -u                            Parse events from userspace.\n"
```